src/main/org/nlogo/prim/plot/primitives.scala

```diff
@@ -317,12 +317,25 @@ final class _setplotpenmode extends PlotCommand(Syntax.NumberType) {
   }
 }
 
-final class _setplotpencolor extends PlotCommand(Syntax.NumberType) {
+final class _setplotpencolor extends PlotCommand(Syntax.NumberType | Syntax.ListType) {
   import org.nlogo.api.Color
   override def perform(context: Context) {
-    currentPen(context).color =
-      Color.getARGBbyPremodulatedColorNumber(
-        Color.modulateDouble(argEvalDoubleValue(context, 0)))
+    var obj = args(0).report(context);
+    obj match {
+      case rgbList: LogoList =>
+        try
+          currentPen(context).color = Color.getARGBIntByRGBAList(rgbList)
+        catch {
+          case e: ClassCastException =>
+            throw new org.nlogo.nvm.EngineException(context, this, displayName + " an rgb list must contain only numbers")
+        }
+      case c: java.lang.Double => 
+        currentPen(context).color =
+          Color.getARGBbyPremodulatedColorNumber(
+          Color.modulateDouble(c))
```

Comment:
Ahh, that makes more sense. So if that one call had multiple lines would I keep it in line  Color.modulateDouble(c)